### ISSUE TITLE
dependency-check formula Error: SHA256 mismatch

### ISSUE THREAD

**Issue by GabLeRoux** [Homebrew/homebrew-core#14310]
```bash
brew update && brew install dependency-check
```
```
Updated 1 tap (homebrew/core).
==> Updated Formulae
svgcleaner
==> Using the sandbox
==> Downloading https://dl.bintray.com/jeremy-long/owasp/dependency-check-1.4.5-release.zip
Already downloaded: /Users/gableroux/Library/Caches/Homebrew/dependency-check-1.4.5.zip
Error: SHA256 mismatch
Expected: 129323b238c372b156c0c315d0b365518025765f3c0855f0ec6ac385f91b91be
Actual: 5ba6934ef5ccefb47b1984448704f86133840b4bb7aaa463ff46dbccdf196dba
Archive: /Users/gableroux/Library/Caches/Homebrew/dependency-check-1.4.5.zip
To retry an incomplete download, remove the file above.
```

Not sure what they changed, but SHA256 mismatch

```bash
brew info dependency-check
```
```
dependency-check: stable 1.4.5
OWASP Dependency Check
https://www.owasp.org/index.php/OWASP_Dependency_Check
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/dependency-check.rb
==> Requirements
Required: java ✔
```

---

- [x] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Homebrew/brew/issues/new. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/caskroom/homebrew-cask/new
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [x] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

```
❯ brew config
HOMEBREW_VERSION: 1.2.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: 54e01d76db34a3a649c79b189873bfd0a52dbc9f
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: bbff1f85d0030bdcecad152ee64ce5fa2bfbcba7
Core tap last commit: 3 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 8.1 build 802
Git: 2.11.0 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Perl: /usr/local/bin/perl => /usr/local/Cellar/perl/5.24.1/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /Users/gableroux/.rbenv/shims/ruby => /Users/gableroux/.rbenv/versions/2.4.0/bin/ruby
Java: 1.8.0_121, 1.7.0_80
macOS: 10.12.5-x86_64
Xcode: 8.3.2
CLT: 8.3.2.0.1.1492020469
X11: N/A
```

**Comment by ilovezfs** [comment]
Reported https://github.com/jeremylong/DependencyCheck/issues/758.

**Comment by jeremylong** [comment]
Re-posting my response to the dependency-check issue [#758](https://github.com/jeremylong/DependencyCheck/issues/758):

No, we were not hacked. Windows Defender was flagging OWASP dependency-check (incorrectly) as malware (see issue [#748](https://github.com/jeremylong/DependencyCheck/issues/748)). In part of my efforts to understand the problem I rebuilt the tagged version (1.4.5) on a clean Ubuntu box and re-uploaded it to bintray - without realizing the downstream consequences to the brew install. No virus was contained in the original or newly uploaded version - I contacted the Windows Defender Support and the virus definitions were updated shortly thereafter.
